types/nightwatch/index.d.ts

```diff
@@ -594,6 +594,12 @@ export interface NightwatchOptions {
      */
     skiptags?: string;
 
+    /**
+	 * Tag(s) used/to be used during test execution.
+	 * Can be a single tag or an array of tags.
+	 */
+	tag_filter?: string | string[];
```

Comment:
It looks like there are a mix of spaces and tabs right now. Try making them all spaces:

```suggestion
    /**
     * Tag(s) used/to be used during test execution.
     * Can be a single tag or an array of tags.
     */
    tag_filter?: string | string[];
```